rdflib/namespace/__init__.py

```diff
@@ -101,6 +101,9 @@ class Namespace(str):
     False
     """
 
+    def __init__(self, *args, **kwargs) -> None:
```

Comment:
You should rather just type `__new__` - I think this will have other runtime consequences and typing new works. I will make a PR against your branch to type `__new__`.

Reply:
There are a couple of other places you also do this, I made a PR https://github.com/ajnelson-nist/rdflib/pull/9 that types `__new__` in all cases instead of adding `__init__`.

Reply:
Agreed, I'm fine not adding the `__init__` methods.  I'll make sure the `Node.__init__()` gets reverted too.